s3fs/core.py

```diff
@@ -296,6 +296,7 @@ def __init__(
         asynchronous=False,
         loop=None,
         max_concurrency=1,
+        fixed_upload_size: bool = False,
```

Comment:
Could use a docstring description, probably explaining that this is required for writing files to R2.

Reply:
sure